### ISSUE TITLE
Improve accessibility SfSegmentedControl (items)

### ISSUE THREAD

**Issue by jfversluis** [syncfusion/maui-toolkit#67]
### Description

As reported on the .NET MAUI repository [here](https://github.com/dotnet/maui/issues/26862), the accessibility of the enclosed items in a SfSegmentedControl is reduced as they cannot be reached by using just the keyboard.

I have selected macOS right now as affected platform, I assume that Windows is affected as well. Not sure about Android and iOS, but might be affected too.

### Steps to Reproduce

(Taken from the issue reported on the .NET MAUI issue linked above)

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter.
4. Observe that Dark/light theme button is not accessible using the keyboard keys.

### Version with bug

1.0.2

### Is this a regression from previous behavior?

No, this is a new issue

### Last Known Working Version

1.0.2

### Affected platforms

macOS
